Tools/CMake/AMReXOptions.cmake

```diff
@@ -157,6 +157,20 @@ cmake_dependent_option( AMReX_DPCPP_ONEDPL "Enable DPCPP's oneDPL algorithms"  O
    "AMReX_GPU_BACKEND STREQUAL SYCL" OFF)
 print_option(  AMReX_DPCPP_ONEDPL )
 
+if (AMReX_DPCPP)
+   set(AMReX_INTEL_ARCH_DEFAULT "IGNORE")
+   if (DEFINED ENV{AMREX_INTEL_ARCH})
+      set(AMReX_INTEL_ARCH_DEFAULT "$ENV{AMREX_INTEL_ARCH}")
+   endif()
+
+   set(AMReX_INTEL_ARCH ${AMReX_INTEL_ARCH_DEFAULT} CACHE STRING
```

Comment:
Let's document `AMReX_INTEL_ARCH` in `Docs/sphinx_documentation/source/GPU.rst`, so users find it :)

Reply:
I will do it.